README.rst

```diff
@@ -87,3 +91,7 @@ Especially the import of extensions can take a while, therefore you can use diff
 - ``./typo3/cli_dispatch.phpsh extbase monitoring:importClients`` to fetch the client data
 
 
+Contribution
```

Comment:
we don't do that, I can write something in release notes as well, pls remove

Reply:
:-[